.github/workflows/manifest-PR.yml

```diff
@@ -11,6 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: handle manifest PR
+        if: >
+          !(contains(github.event.pull_request.title, 'manifest-pr-skip') ||
+            contains(github.event.pull_request.body, 'manifest-pr-skip'))
```

Comment:
Hopefully it works, but I don't know how to verify it in advance.

Reply:
@57300 . I believe this should rather be fixed in the action itself, so that it also works for other users of this action. @karhama , do you agree?

Reply:
Yes I would recommend changing action itself if cancelled status is causing problems. However I did not fully understand in which case cancelled becomes the problem?  Idea in action side was roughly : success == manifest PR created, failure == failed to create manifest PR (when it should have been created) and  cancelled == "neutral status" i.e. no need to create manifest PR.

Reply:
Updating the action itself sounds reasonable. I'm just unsure what's the best way to do it, seems to me like it could make the code more convoluted.

The only problem with the cancel is that it makes the PR red, which is misleading. It also shows up when filtering PRs by "status:failure". I think a code owner would still be able to merge it though, so I guess it's minor.